opnilog-inference-service/opnilog_trainer.py

```diff
@@ -11,21 +11,66 @@
 from botocore.client import Config
 from const import (
     DEFAULT_MODEL_NAME,
+    ES_ENDPOINT,
+    ES_PASSWORD,
+    ES_USERNAME,
     LOGGING_LEVEL,
     S3_ACCESS_KEY,
     S3_BUCKET,
+    S3_ENDPOINT,
     S3_SECRET_KEY,
     TRAINING_DATA_PATH,
 )
+from elasticsearch import AsyncElasticsearch
+from masker import LogMasker
 from opni_nats import NatsWrapper
 from opnilog_parser import LogParser
 
 logging.basicConfig(format="%(asctime)s - %(levelname)s - %(message)s")
 logger = logging.getLogger(__file__)
 logger.setLevel(LOGGING_LEVEL)
+masker = LogMasker()
+
+es_instance = AsyncElasticsearch(
+    [ES_ENDPOINT],
+    port=9200,
+    http_compress=True,
+    http_auth=(ES_USERNAME, ES_PASSWORD),
+    verify_certs=False,
+    use_ssl=True,
+)
+
+
+async def get_all_training_data(payload):
```

Comment:
what's the difference between this function and the ones in `training-controler`?